code/app/build.gradle

```diff
@@ -1,17 +1,15 @@
-/*
- Copyright 2021 Adobe. All rights reserved.
- This file is licensed to you under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License. You may obtain a copy
- of the License at http://www.apache.org/licenses/LICENSE-2.0
- Unless required by applicable law or agreed to in writing, software distributed under
- the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- OF ANY KIND, either express or implied. See the License for the specific language
- governing permissions and limitations under the License.
- */
-
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
+apply plugin: "com.diffplug.spotless"
+
+spotless {
+    kotlin {
+        target "src/*/java/**/*.kt"
+        ktlint('0.41.0')
```

Comment:
I decided to use ktlint here to format the Kotlin test app. There were minimal changes to the app files after running the formatting.